src/engine/Engine.ts

```diff
@@ -232,8 +232,8 @@ export class Engine extends Class {
     * Default [[IEngineOptions]]
     */
    private static _DefaultEngineOptions: IEngineOptions = {
-      width: 0,
-      height: 0,
+      canvasWidth: 0,
```

Comment:
@rehlert95 @jedeen I think we have consensus on renaming `getWidth()` and `getHeight()` `getDrawWidth()` and `getDrawHeight()` as well as not changing `IEngineOptions`